### ISSUE TITLE
Bug in get_contrast function for sequences

### ISSUE THREAD

**Issue by SWeinmueller** [MRsources/MRzero-Core#46]
https://github.com/MRsources/MRzero-Core/blob/ae55cee6f85da68c379f7583d4376e744a93cfa7/python/MRzeroCore/sequence.py#L392C1-L394C10

For dummys the adc_usage has to be > 0!

return torch.cat(
            [rep.adc_usage[rep.adc_usage > 0] == contrast for rep in self]
        )

**Comment by J-Endres** [comment]
I don't fully understand the bug yet - are you using adc_usage = -1 or similar for dummy repetitions and don't want it to show up in the contrast masks?

**Comment by SWeinmueller** [comment]
Dummy repetitions has adc_usage = -1 for exporter reason and they should not show up in contrast mask, otherwise the dimension of the signal and the contrast mask don't match
